docs/api_reference/src/quantities.tex

```diff
@@ -913,11 +913,22 @@
 a specialization of \tcode{power<$x$, $n$, $d$>}, then its input value is $x^{n/d}$, and
 the following also apply.
 \item
-Otherwise, a type $x$ is an input constant.
+Otherwise, a type $x$ is an input symbolic constant.
 \end{itemize}
 \begin{example}
-The input of \tcode{derived_unit<si::kilo_<si::metre>, per<power<non_si::hour, 2>>>}
-is $\tcode{km}/\tcode{h}^2$.
+Item by item, this algorithm step goes from the \Cpp{} parameter type
+\tcode{derived_unit<si::kilo_<si::metre>, per<power<non_si::hour, 2>>}
+\begin{itemize}
+\item
+to $\tcode{si::kilo_<si::metre>} \times \tcode{per<power<non_si::hour, 2>}$ (product of \tcode{To}'s arguments),
+\item
+to $\tcode{si::kilo_<si::metre>} \times 1/\tcode{power<non_si::hour, 2>}$ (product of \tcode{per}'s arguments as denominator),
+\item
+to $\tcode{si::kilo_<si::metre>} \times 1/\tcode{non_si::hour}^2$ (\tcode{power}s as powers),
+\item
+to $x \times 1/y^2$ where $x = \tcode{si::kilo_<si::metre>}$ and $y = \tcode{non_si::hour}$ (symbolic substitution)
+in the mathematical domain.
```

Comment:
Are those `x` and `y` the same things as in the wording? If not, maybe it is  better to rename them, for example, to `n` and `d` (nominator/denominator) or `a` and `b`?